profiles/package.mask

```diff
@@ -33,6 +33,10 @@
 
 #--- END OF EXAMPLES ---
 
+# Volkmar W. Pogatzki <gentoo@pogatzki.net> (2022-12-06)
+# Unused Java library. Removal on 2023-01-06.
+dev-java/tomcat-native
+
```

Comment:
it's not a library, it's an add-on that adds binding from tomcat to openssl so that tomcat can use openssl. it's still maintained by upstream and it's still needed. we just lack on bumping it because the new version 2 seems to depend on java 11+ and it would not work with tomcat 9 and 10.0 which still support java 8, so we might need slotting for tomcat-native in fact i guess. also, i was not able to test tomcat-native 2 as it needs openssl-3 which i don't have in my system yet (it's still masked). you can find more info at https://github.com/gentoo/gentoo/pull/26513